How to use line-length in command line for ruff format?

Plain simple, I want to use ruff format and I don't have any toml configuration.

```bash
echo 'aa = [ "blablabla", "blablabla","blablabla","blablabla","blablabla","blablabla","blablabla","blablabla","blablabla","blablabla"]' > test.py
ruff check test.py --select ALL --line-length=120

warning: `one-blank-line-before-class` (D203) and `no-blank-line-before-class` (D211) are incompatible. Ignoring `one-blank-line-before-class`.
warning: `multi-line-summary-first-line` (D212) and `multi-line-summary-second-line` (D213) are incompatible. Ignoring `multi-line-summary-second-line`.
test.py:1:1: D100 Missing docstring in public module
test.py:1:121: E501 Line too long (128 > 120)
Found 2 errors.
(```

But

```bash
ruff format test.py --line-length=120

error: unexpected argument '--line-length' found

  tip: to pass '--line-length' as a value, use '-- --line-length'

Usage: ruff format <FILES|--check|--diff|--config <CONFIG>|--no-cache|--cache-dir <CACHE_DIR>|--respect-gitignore|--no-respect-gitignore|--exclude <FILE_PATTERN>|--force-exclude|--no-force-exclude|--isolated|--stdin-filename <STDIN_FILENAME>|--target-version <TARGET_VERSION>|--preview|--no-preview>

For more information, try '--help'.
```

If using `ruff format test.py` it will use the default 88.

--- a/crates/ruff_cli/src/args.rs
+++ b/crates/ruff_cli/src/args.rs
@@ -409,6 +409,9 @@ pub struct FormatCommand {
     force_exclude: bool,
     #[clap(long, overrides_with("force_exclude"), hide = true)]
     no_force_exclude: bool,
+    /// Set the line-length.
+    #[arg(long, help_heading = "Rule configuration", hide = true)]
+    pub line_length: Option<LineLength>,
     /// Ignore all configuration files.
     #[arg(long, conflicts_with = "config", help_heading = "Miscellaneous")]
     pub isolated: bool,
@@ -552,6 +555,7 @@ impl FormatCommand {
                 stdin_filename: self.stdin_filename,
             },
             CliOverrides {
+                line_length: self.line_length,
                 respect_gitignore: resolve_bool_arg(
                     self.respect_gitignore,
                     self.no_respect_gitignore,

--- a/crates/ruff_cli/src/args.rs
+++ b/crates/ruff_cli/src/args.rs
@@ -410,7 +410,7 @@ pub struct FormatCommand {
     #[clap(long, overrides_with("force_exclude"), hide = true)]
     no_force_exclude: bool,
     /// Set the line-length.
-    #[arg(long, help_heading = "Rule configuration", hide = true)]
+    #[arg(long, help_heading = "Format configuration")]
     pub line_length: Option<LineLength>,
     /// Ignore all configuration files.
     #[arg(long, conflicts_with = "config", help_heading = "Miscellaneous")]

--- a/crates/ruff_python_formatter/README.md
+++ b/crates/ruff_python_formatter/README.md
@@ -117,10 +117,11 @@ quote-style = "single"
 ```
 
 The Ruff formatter also respects Ruff's [`line-length`](https://docs.astral.sh/ruff/settings/#line-length)
-setting, which also can be provided via a `pyproject.toml` or `ruff.toml` file.
+setting, which also can be provided via a `pyproject.toml` or `ruff.toml` file, or on the CLI, as
+in:
 
-```toml
-line-length = 80
+```console
+ruff format --line-length 100 /path/to/file.py
 ```
 
 ### Excluding code from formatting

--- a/docs/configuration.md
+++ b/docs/configuration.md
@@ -402,6 +402,9 @@ File selection:
       --exclude <FILE_PATTERN>  List of paths, used to omit files and/or directories from analysis
       --force-exclude           Enforce exclusions, even for paths passed to Ruff directly on the command-line. Use `--no-force-exclude` to disable
 
+Format configuration:
+      --line-length <LINE_LENGTH>  Set the line-length
+
 Log levels:
   -v, --verbose  Enable verbose logging
   -q, --quiet    Print diagnostics, but nothing else

